client/cluster.go

```diff
@@ -406,6 +432,16 @@ func (c *ClusterClient) ResetNode(name string) error {
 		return err
 	}
 
+	if machine.NodeType == api.Master {
+		c.logger.Debug("client_node_drain_master_lb_timeout_wait")
+
+		// Wait time based on the longest load balancer health check timeout,
+		// which is CityCloud.
+		// TODO: Find a better solution for this. Could we perhaps force remove
+		// load balancer members?
+		time.Sleep((10*5 + 20*4) * time.Second)
```

Comment:
Directly manipulating the load balancers would be really cool! I guess that would require specific implementations for each cloud provider though. Maybe a feature to consider in the future? :slightly_smiling_face:

Reply:
https://github.com/elastisys/ck8s-cluster/issues/46